kitchen-google.gemspec

```diff
@@ -1,26 +1,25 @@
 # -*- coding: utf-8 -*-
 Gem::Specification.new do |s|
-  s.name        = 'kitchen-google'
-  s.version     = '0.3.0'
-  s.date        = '2016-01-23'
-  s.summary     = 'Kitchen::Driver::Gce'
-  s.description = 'A Test-Kitchen driver for Google Compute Engine'
-  s.authors     = ['Andrew Leonard']
-  s.email       = 'andy@hurricane-ridge.com'
-  s.files       = `git ls-files`.split($/) # rubocop:disable SpecialGlobalVars
-  s.homepage    = 'https://github.com/test-kitchen/kitchen-google'
-  s.license     = 'Apache 2.0'
+  s.name        = "kitchen-google"
+  s.version     = "0.3.0"
+  s.date        = "2016-01-23"
+  s.summary     = "Kitchen::Driver::Gce"
+  s.description = "A Test-Kitchen driver for Google Compute Engine"
+  s.authors     = ["Andrew Leonard", "Chef Partner Engineering"]
+  s.email       = ["andy@hurricane-ridge.com", "partnereng@chef.io"]
```

Comment:
You're not the first. We actually released a blog post with our team's alias as "partnering" recently. So, we picked a bad alias. Go team!